frontend/src/metabase/dashboard/components/Dashboard/Dashboard.jsx

```diff
@@ -121,28 +53,15 @@ class Dashboard extends Component {
       : null;
   }
 
-  throttleParameterWidgetStickiness = _.throttle(
-    () => updateParametersWidgetStickiness(this),
-    SCROLL_THROTTLE_INTERVAL,
-  );
-
-  // NOTE: all of these lifecycle methods should be replaced with DashboardData HoC in container
   async componentDidMount() {
     await this.loadDashboard(this.props.dashboardId);
-
-    const main = getMainElement();
-    main.addEventListener("scroll", this.throttleParameterWidgetStickiness, {
-      passive: true,
-    });
-    main.addEventListener("resize", this.throttleParameterWidgetStickiness, {
-      passive: true,
-    });
   }
 
   async componentDidUpdate(prevProps) {
+    updateParametersWidgetStickiness(this);
```

Comment:
when component is updated, we recalculate stickiness

Reply:
Why don't we do this in `if (prevProps.dashboardId !== this.props.dashboardId)` as before? To me, it's more straightforward to replace `this.throttleParameterWidgetStickiness()` with `updateParametersWidgetStickiness(this)`

Reply:
I had in mind an edge case when you're on mobile and new parameter is added, but we don't allow to edit dashboards on mobile 🤔 

maybe you're right, I just wanted to be on the safe side as it's a very cheap check atm

Reply:
I got one test broken (with the tall card), reverted changes to my initial variant